src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerCreateForm.tsx

```diff
@@ -84,6 +92,12 @@ export const TemplatesCatalogDrawerCreateForm: FC<TemplatesCatalogDrawerCreateFo
                       value={nameField}
                     />
                   </FormGroup>
+                  <FormGroupHelperText validated={vmNameValidated}>
+                    {vmNameValidated === ValidatedOptions.error &&
+                      t('Maximum name length is {{ maxNameLength }} characters', {
+                        maxNameLength: MAX_K8S_NAME_LENGTH,
+                      })}
+                  </FormGroupHelperText>
```

Comment:
This also repeats so can be in a different shared component

Reply:
why u arent using getHelperTextMessage ?

Reply:
Fixed!